---
title: "MICB 425 portfolio- Microbial Ecological Genomics: From Genomes to Biomes, Spring 2018"
author: "Haya Abuzuluf 32601130"
date: "version `r format(Sys.time(), '%B %d, %Y')`"
output:
  html_document:
    toc: true
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```

# Module 01


## Module 01 portfolio check

* Installation check
    + Completion status:
    + Comments:
* Portfolio repo setup
    + Completion status: 
    + Comments:
* RMarkdown Pretty html Challenge
    + Completion status:
    + Comments:
* Evidence worksheet_01
    + Completion status:
    + Comments:
* Evidence worksheet_02
    + Completion status:
    + Comments:
* Evidence worksheet_03
    + Completion status:
    + Comments:
* Problem Set_01
    + Completion status:
    + Comments:
* Problem Set_02
    + Completion status:
    + Comments:
* Writing assessment_01
    + Completion status:
    + Comments:
* Additional Readings
    + Completion status:
    + Comments  
    
## Data science Friday  


### Installation check


![](1.png)

![](2.png)  

![](3.png)  

### Portfolio repo setup
Detail the code you used to create, initialize, and push your portfolio repo to GitHub. This will be helpful as you will need to repeat many of these steps to update your porfolio throughout the course.


`git add .`  
`git commit -m "type message"`  
`git push`  

### RMarkdown pretty html challenge
Paste your code from the in-class activity of recreating the example html.

* [R Markdown PDF Challenge] [R Markdown PDF Challenge ]
    + [Challenge Goals] [Challenge Goals]
        + [Here's a header!] [Here's a header!]  
        + [Table Time]  [Table Time]

# R Markdown PDF Challenge 

The following assignment is an exercise for the reproduction of this .html document using the RStudio and RMarkdown tools we’ve shown you in class. Hopefully by the end of this, you won’t feel at all the way this poor PhD student does. We’re here to help, and when it comes to R, the internet is a really valuable resource. This open-source program has all kinds of tutorials online.  

![](phd011718s.gif)  
http://phdcomics.com/ Comic posted 1-17-2018  

## Challenge Goals  

The goal of this R Markdown html challenge is to give you an opportunity to play with a bunch of different RMarkdown formatting. Consider it a chance to flex your RMarkdown muscles. Your goal is to write your own RMarkdown that rebuilds this html document as close to the original as possible. So, yes, this means you get to copy my irreverant tone exactly in your own Markdowns. It’s a little window into my psyche. Enjoy =)  

__hint: go to the [PhD Comics website](www.phdcomics.com) to see if you can find the image above__  
_If you can’t find that exact image, just find a comparable image from the PhD Comics website and include it in your markdown_  

### Here's a header!  

Let’s be honest, this header is a little arbitrary. But show me that you can reproduce headers with different levels please. This is a level 3 header, for your reference (you can most easily tell this from the table of contents)

#### Another header, now with maths  

Perhaps you’re already really confused by the whole markdown thing. Maybe you’re so confused that you’ve forgotton how to add. Never fear! A ~~calculator~~ R is here:

```{r}
1231521+12341556280987
```

```{r echo=FALSE}
## [1] 1.234156e+13
```

### Table Time

Or maybe, after you’ve added those numbers, you feel like it’s about time for a table!  
I’m going to leave all the guts of the coding here so you can see how libraries (R packages) are loaded into R (more on that later). It’s not terribly pretty, but it hints at how R works and how you will use it in the future. The summary function used below is a nice data exploration function that you may use in the ^future^.

```{r}
library(knitr)
kable(summary(cars),caption="I made this table with kable in the knitr package library")
```

And now you’ve almost finished your first RMarkdown! Feeling excited? We are! In fact, we’re so excited that maybe we need a big finale eh? Here’s ours! Include a fun gif of your choice!

![](science.gif)




## Origins and Earth Systems
### Evidence worksheet 01 

[Whitman *et al* 1998](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC33863/)

#### Learning objectives
Describe the numerical abundance of microbial life in relation to ecology and biogeochemistry of Earth systems. 

#### General questions
* What were the main questions being asked?  

Number of prokaryotes?  
How do prokaryotes affect the global carbon cycle?  
Genetic diversity and number of mutations?  


* What were the primary methodological approaches used?  

Referenced papers and estimated numbers based on assumptions

* Summarize the main results or findings.  

Total carbon of prokaryotes is 60-100% of carbon.  
Cellular production is highest.  
More mutations in oceans than other populations (high turnover time, more mutations).  


* Do new questions arise from the results?  

Turnover time at subsurface?  
How is the bacterial population vs archaea population (distinguish between the two)?  
Species definition of prokaryotes vs eukaryotes  


* Were there any specific challenges or advantages in understanding the paper (*e.g.* did the authors provide sufficient background information to understand experimental logic, were methods explained adequately, were any specific assumptions made, were conclusions justified based on the evidence, were the figures or tables useful and easy to understand)?  

Not enough background information, they just jump into assumptions and calculations.  
Tables and figures are not summarized or explained in results.  

### Problem set 01
#### Learning objectives: 
Describe the numerical abundance of microbial life in relation to the ecology and biogeochemistry of Earth systems.

#### Specific questions:
* What are the primary prokaryotic habitats on Earth and how do they vary with respect to their capacity to support life? Provide a breakdown of total cell abundance for each primary habitat from the tables provided in the text.  

* What is the estimated prokaryotic cell abundance in the upper 200 m of the ocean and what fraction of this biomass is represented by marine cyanobacterium including Prochlorococcus? What is the significance of this ratio with respect to carbon cycling in the ocean and the atmospheric composition of the Earth?  

* What is the difference between an autotroph, heterotroph, and a lithotroph based on information provided in the text?  

* Based on information provided in the text and your knowledge of geography what is the deepest habitat capable of supporting prokaryotic life? What is the primary limiting factor at this depth?  

* Based on information provided in the text your knowledge of geography what is the highest habitat capable of supporting prokaryotic life? What is the primary limiting factor at this height?  

* Based on estimates of prokaryotic habitat limitation, what is the vertical distance of the Earth’s biosphere measured in km?  

* How was annual cellular production of prokaryotes described in Table 7 column four determined? (Provide an example of the calculation)  

* What is the relationship between carbon content, carbon assimilation efficiency and turnover rates in the upper 200m of the ocean? Why does this vary with depth in the ocean and between terrestrial and marine habitats?  

* How were the frequency numbers for four simultaneous mutations in shared genes determined for marine heterotrophs and marine autotrophs given an average mutation rate of 4 x 10^-7^ per DNA replication? (Provide an example of the calculation with units. Hint: cell and generation cancel out)  

* Given the large population size and high mutation rate of prokaryotic cells, what are the implications with respect to genetic diversity and adaptive potential? Are point mutations the only way in which microbial genomes diversify and adapt?  

* What relationships can be inferred between prokaryotic abundance, diversity, and metabolic potential based on the information provided in the text?  


### Evidence Worksheet_02 “Life and the Evolution of Earth’s Atmosphere”
#### Learning objectives: 
Comment on the emergence of microbial life and the evolution of Earth systems
    
    
* Indicate the key events in the evolution of Earth systems at each approximate moment in the time series. If times need to be adjusted or added to the timeline to fully account for the development of Earth systems, please do so. 

    + 4.6 billion years ago  
    
    + 4.2 billion years ago  
    
    + 3.8 billion years ago  
    
    + 3.75 billion years ago  
    
    + 3.5 billion years ago  
    
    + 3.0 billion years ago  
    
    + 2.7 billion years ago  
    
    + 2.2 billion years ago  
    
    + 2.1 billion years ago
    
    + 1.3 billion years ago
    
    + 550,000 years ago
    
    + 200,000 years ago


* Describe the dominant physical and chemical characteristics of Earth systems at the following waypoints:  

    + Hadean  
    
    + Archean  
    
    + Precambrian  
    
    + Proterozoic  
    
    + Phanerozoic  
    







### Problem set_02 “Microbial Engines”
#### Learning objectives: 
Discuss the role of microbial diversity and formation of coupled metabolism in driving global biogeochemical cycles.

#### Specific Questions:
* What are the primary geophysical and biogeochemical processes that create and sustain conditions for life on Earth? How do abiotic versus biotic processes vary with respect to matter and energy transformation and how are they interconnected?  

* Why is Earth’s redox state considered an emergent property?
  

* How do reversible electron transfer reactions give rise to element and nutrient cycles at different ecological scales? What strategies do microbes use to overcome thermodynamic barriers to reversible electron flow?  

* Using information provided in the text, describe how the nitrogen cycle partitions between different redox “niches” and microbial groups. Is there a relationship between the nitrogen cycle and climate change?   

* What is the relationship between microbial diversity and metabolic diversity and how does this relate to the discovery of new protein families from microbial community genomes?  

* On what basis do the authors consider microbes the guardians of metabolism?  






### Module 01 references



Whitman WB, Coleman DC, and Wiebe WJ. 1998. Prokaryotes: The unseen majority. Proc Natl Acad Sci USA. 95(12):6578–6583.  [PMC33863](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC33863/)



# Module 02  

## Data science Friday  

## Remapping the body of the world


### Evidence worksheet_03

### Problem set_03 “Metagenomics: Genomic Analysis of Microbial Communities”  
#### Learning objectives:   
Specific emphasis should be placed on the process used to find the answer. Be as comprehensive as possible e.g. provide URLs for web sources, literature citations, etc.  
*(Reminders for how to format links, etc in RMarkdown are in the RMarkdown Cheat Sheets)*

#### Specific Questions:
* How many prokaryotic divisions have been described and how many have no cultured representatives (microbial dark matter)?
  

* How many metagenome sequencing projects are currently available in the public domain and what types of environments are they sourced from?
  

* What types of on-line resources are available for warehousing and/or analyzing environmental sequence information (provide names, URLS and applications)?  

* What is the difference between phylogenetic and functional gene anchors and how can they be used in metagenome analysis?   

* What is metagenomic sequence binning? What types of algorithmic approaches are used to produce sequence bins? What are some risks and opportunities associated with using sequence bins for metabolic reconstruction of uncultivated microorganisms?  

* Is there an alternative to metagenomic shotgun sequencing that can be used to access the metabolic potential of uncultivated microorganisms? What are some risks and opportunities associated with this alternative?  

 